src/functions/slack-api/handle-events.js

```diff
@@ -22,6 +22,14 @@ botApp.event(events.TEAM_JOIN_EVENT, async ({ event, logger }) => {
     }
 });
 
+botApp.event(events.APP_MENTION, async ({ event, logger }) => {
+    try {
```

Comment:
When you use promises, try to avoid try/catch block. Use error handlers instead.